docker/Makefile

```diff
@@ -34,7 +57,7 @@ endif
 test: prep
 	@bundle install --path $$BUNDLE_PATH --gemfile $$GEMFILE
 	@PUPPET_TEST_DOCKER_IMAGE=$(NAMESPACE)/puppet-bolt:$(version) \
-		bundle exec --gemfile $$GEMFILE rspec puppet-bolt/spec
+		bundle exec rspec puppet-bolt/spec
```

Comment:
still need `--gemfile $$GEMFILE` here, but otherwise looks good!

Reply:
The version of bundler on the worker node does not seem to support that :( 
```
13:52:37 bundler: command not found: --gemfile
13:52:37 Install missing gem executables with `bundle install`
13:52:37 Makefile:58: recipe for target 'test' failed
```
https://jenkins-master-prod-1.delivery.puppetlabs.net/view/__experimental%20automatic/job/experimental_auto_ship-bolt_build_and_push_bolt_docker_image/17/console

Reply:
huh, weird.

I think this should be fine, but if you end up setting container builds up on travis and run into weird issues there we may need to revisit this.

Reply:
Sounds good. Thanks for the review!